vars/notify.groovy

```diff
@@ -281,6 +281,67 @@ void mattermost(Map config = [:]) {
   im.mattermost((NOTIFY_MATTERMOST): buildResultConfig)
 }
 
+/**
+ * Sends a Microsoft Teams notification using the Office365 Connector plugin
+ *
+ * @param config The configuration for the step
+ */
+void teams(Map config = [:]) {
+
+  Logger log = new Logger("notify.teams")
+
+  NotificationTriggerHelper triggerHelper = this.getTriggerHelper()
+  String defaultColor = triggerHelper.getTrigger().getColor()
+
+  Map defaultConfig = [
+    (NOTIFY_TEAMS): [
+      (MAP_MERGE_MODE)                           : (MapMergeMode.REPLACE),
+      (NOTIFY_TEAMS_ENABLED)                     : true,
```

Comment:
Code formatting,  see the x position of the double colon. Please format the code.

Reply:
check ✔️